From: Christian Borntraeger <borntraeger@de.ibm.com>
Subject: vmcp: Missing man page
Patch-mainline: n/a
Git-commit: -
References: bnc#825037, LTC#94453

Description:  vmcp: Missing man page 
Symptom:      make D=1 shows that for vmcp the message description is missing.
Problem:      No message description for vmcp.
Solution:     Provide dummy description.
Reproduction: -

Signed-off-by: Christian Borntraeger <borntraeger@de.ibm.com>
Acked-by: John Jolly <jjolly@suse.de>
---
 Documentation/kmsg/s390/vmcp |    1 +
 1 file changed, 1 insertion(+)

--- /dev/null
+++ b/Documentation/kmsg/s390/vmcp
@@ -0,0 +1 @@
+/*? Text: "The z/VM CP interface device driver cannot be loaded without z/VM\n" */
